screens/VerifyEmailScreen.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from "react";
+import { SafeAreaView, StyleSheet } from "react-native";
+import * as Yup from "yup";
+
+import AppButton from "../components/AppButton";
+import AppFormField from "../components/AppFormField";
+import runCrypto from "../auth/crypto-hashing";
+import defaultStyles from "../config/defaultStyles";
+import ErrorMessage from "../components/ErrorMessage";
+import Form from "../components/Form";
+import AuthContext from "../auth/Context";
+import SubmitButton from "../components/SubmitButton";
+import SuccessMessage from "../components/SuccessMessage";
+
+const validationSchema = Yup.object().shape({
+  email: Yup.string().required().email().label("Email"),
+});
+
+export default function RegisterPassword({ navigation }) {
```

Comment:
This is a function component but I believe we're going with class-based components. Also, wrong name?

Reply:
Nevermind - but still same comment re. the name of the component.